src/Libraries/DynamoUnits/Properties/AssemblyInfo.cs

```diff
@@ -11,4 +11,9 @@
 // The following GUID is for the ID of the typelib if this project is exposed to COM
 [assembly: Guid("2a29d5d1-ece3-4802-a758-562fec421051")]
 [assembly:InternalsVisibleTo("DynamoCoreTests")]
-[assembly: InternalsVisibleTo("DynamoCoreWpfTests")]
\ No newline at end of file
+[assembly: InternalsVisibleTo("DynamoCoreWpfTests")]
+[assembly: InternalsVisibleTo("UnitsNodeModels")]
+
+
+//move DynamoUnits.Display to DynamoCore.dll
+[assembly: TypeForwardedTo(typeof(DynamoUnits.Display))]
```

Comment:
so we are keeping this assembly (DynamoUnits) around just for type forwarding right ?

Reply:
sorry this change is a bit confusing.
So the old DynamoUnits project will now be UnitsCore.csproj but it will produce the same dll as before (for compatibility sake and typeforwarding)
Is that correct ?

Reply:
hey @pinzart90 it's pretty confusing yes, let me try to explain.

`DynamoUnits.dll` (previously Units.csproj) contains the new ZT nodes that use the Forge types. It also exposes those Forge types with Dynamo wrappers to users can write ZT nodes that accept and return those types. For example `DynamoUnits.Unit`.

I renamed this csproj to `UnitsCore` - but I have no desire to mess with the assembly name at this time for the reasons you mentioned - lets do it at 2.20 or 3.0 etc.

`UnitsUI.dll` contained both the WPF controls and the node models nodes for some more complex UI units nodes. I've split this into 
`UnitsUI.dll` and `UnitsNodeModels.dll`

I originally put the nodeModel nodes into DynamoUnits.dll - but these nodes have dependencies on `CoreNodeModels` and I didn't want to force all ZT node authors to reference/pull those packages just to get access to the core `Unit` type for their nodes.

I hope that explanation helps.